iis-wf/iis-wf-primary/src/main/resources/eu/dnetlib/iis/wf/primary/main/oozie_app/workflow.xml

```diff
@@ -490,6 +490,10 @@
                     <name>output_faults</name>
                     <value>${workingDir}/primary_import/faults</value>
                 </property>
+                <property>
+                    <name>output_report</name>
+                    <value>${workingDir}/report</value>
```

Comment:
My intention was to store all these reports in the same parent folder.
Currently report for IS import will be located under `${workingDir}/report/import_info_space`. And for example report for datasets import can be located under `${workingDir}/report/import_dataset`.
This way joining all the partial reports can be located somewhere after export and we can provide just one parameter with this parent input location.
Enabling possibility to set partial report locations to "anywhere we want" causes a lot of juggling with path parameters.
In my opinion this is unnecessary.
We always use param `output_report` in workflows that reports something.
In primary workflow we only pass that `output_report` for `import`, `processing`, `export` subworkflows which is parent folder for all reports and let the `<propagate-configuration />` to do the rest. We don't have to modify anyhow middle-part workflows. And using only one param to control all the partial reports.